cogs/web.py

```diff
@@ -4,16 +4,17 @@
 import aiohttp
 import discord
 import requests
+from discord.ext import commands, tasks
 from discord import app_commands
 from pymongo import MongoClient
 
 # database connection
 cluster = MongoClient(os.environ.get("mongo"))
 db = cluster["discord"]
-collection = db["status"]
+collection = db["status-web"]
```

Comment:
The collection in mongo is called "status" not "status-web". Changing this would require that be changed in the database, which it can't (easily)